Tests/CommonTests/Extensions/URLExtensionTests.swift

```diff
@@ -21,6 +21,14 @@ import XCTest
 
 final class URLExtensionTests: XCTestCase {
 
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+
+        if #available(macOS 14, *) {
+            throw XCTSkip("This test can't run on macOS 14 or higher")
+        }
```

Comment:
That's interesting, what's up with this?

Reply:
CI is still using macOS 13.  Running those tests locally fails.